modules/ballerina-core/src/main/java/org/ballerinalang/connector/impl/BServerConnectorFuture.java

```diff
@@ -32,23 +31,19 @@
 public class BServerConnectorFuture implements ConnectorFuture {
     private ConnectorFutureListener connectorFutureListener;
 
-    private BValue[] response;
     private BallerinaException ex;
     private boolean success = false;
 
     @Override
     public void setConnectorFutureListener(ConnectorFutureListener futureListener) {
         this.connectorFutureListener = futureListener;
-        if (response != null) {
-            connectorFutureListener.notifyReply(response);
-        } else if (ex != null) {
+        if (ex != null) {
```

Comment:
Instead of saying "ex" shall we say exception.. long names are better than enigmatic names.

Reply:
ack